apps/server/src/modules/board/repo/board-node.builder-impl.ts

```diff
@@ -84,6 +86,19 @@ export class BoardNodeBuilderImpl implements BoardNodeBuilder {
 		this.registerNode(textElementNode);
 	}
 
+	buildFileElementNode(fileElement: FileElement, parentId?: string | undefined, position?: number | undefined): void {
```

Comment:
I am surprised that position comes from the outside. Position is depending on the parent so I expected it to be derived from the parent.

Reply:
Good point. We did the refactoring in `BC-3687`:
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4048/commits/8cf9bfce75e65f77bff7e40baa8148bbd0eef7d9